README.md

```diff
@@ -133,8 +174,9 @@ cd zax
 . tools/relay
 . tools/redis # new console window
 ```
+
 ## Demo
-To see Glow and Zax in action, check out the [Live Demo](https://zax_test.vault12.com). This is a test project included in Zax called [Zax-Dash](https://github.com/vault12/zax-dash).
+To see Glow and Zax in action, check out the [Live Demo](https://zax-test.vault12.com). This is a test project included in Zax called [Zax-Dash](https://github.com/vault12/zax-dash).
```

Comment:
Did you want to change this url to z.vault12.com?

Reply:
`zax-test` is live, we had to discard `zax_test` because of "let's encrypt" bizarre limit.